include/Megamix/Hooks.hpp

```diff
@@ -7,10 +7,10 @@ namespace Megamix::Hooks {
     void TickflowHooks();
     void TempoHooks();
     void RegionHooks();
+    void CommandHook();
     void DisableAllHooks();
 
     template<typename T> T StubbedFunction();
     template<typename T> void StubFunction(u32 address);
 }
```

Comment:
same thing as the Commands.hpp comment (although the final line break missing is my bad)